front/app/containers/LandingPage/SignedInHeader.tsx

```diff
@@ -354,8 +352,8 @@ class SignedInHeader extends PureComponent<Props, State> {
           id="hook-header"
         >
           <ScreenReaderOnly>
-            {headerTitleMultiLoc ? (
```

Comment:
Unclear why this value was in the screen reader and not the actual signed-in multiloc. There's also a fallback in `genericTitle` so I think this can be removed

Reply:
Let's continue it in slack? https://citizenlabco.slack.com/archives/CQZGXTFGD/p1659472748210379?thread_ts=1659429442.979059&cid=CQZGXTFGD